diff --git a/module/spl/Info.plist b/module/spl/Info.plist
index 531f4db..6e09d77 100644
--- a/module/spl/Info.plist
+++ b/module/spl/Info.plist
@@ -29,9 +29,9 @@
 	<key>DTPlatformVersion</key>
 	<string>GM</string>
 	<key>DTSDKBuild</key>
-	<string>12F37</string>
+	<string>14A383</string>
 	<key>DTSDKName</key>
-	<string>macosx10.8</string>
+	<string>macosx10.10</string>
 	<key>DTXcode</key>
 	<string>0620</string>
 	<key>DTXcodeBuild</key>
diff --git a/module/spl/spl-osx.c b/module/spl/spl-osx.c
index 36eebc8..f4969b2 100644
--- a/module/spl/spl-osx.c
+++ b/module/spl/spl-osx.c
@@ -198,7 +198,7 @@ extern kern_return_t _stop(kmod_info_t *ki, void *data);
 __private_extern__ kern_return_t spl_start(kmod_info_t *ki, void *data);
 __private_extern__ kern_return_t spl_stop(kmod_info_t *ki, void *data);
 
-__attribute__((visibility("default"))) KMOD_EXPLICIT_DECL(net.lundman.spl, "1.0.0", _start, _stop)
+__attribute__((visibility("default"))) KMOD_EXPLICIT_DECL(net.lundman.spl, "5.0.10", _start, _stop)
 __private_extern__ kmod_start_func_t *_realmain = spl_start;
 __private_extern__ kmod_stop_func_t *_antimain = spl_stop;
 __private_extern__ int _kext_apple_cc = __APPLE_CC__ ;
