diff --git a/module/spl/spl-osx.c b/module/spl/spl-osx.c
index 36eebc8..8068117 100644
--- a/module/spl/spl-osx.c
+++ b/module/spl/spl-osx.c
@@ -198,7 +198,7 @@ extern kern_return_t _stop(kmod_info_t *ki, void *data);
 __private_extern__ kern_return_t spl_start(kmod_info_t *ki, void *data);
 __private_extern__ kern_return_t spl_stop(kmod_info_t *ki, void *data);
 
-__attribute__((visibility("default"))) KMOD_EXPLICIT_DECL(net.lundman.spl, "1.0.0", _start, _stop)
+__attribute__((visibility("default"))) KMOD_EXPLICIT_DECL(net.lundman.spl, "5.0.8", _start, _stop)
 __private_extern__ kmod_start_func_t *_realmain = spl_start;
 __private_extern__ kmod_stop_func_t *_antimain = spl_stop;
 __private_extern__ int _kext_apple_cc = __APPLE_CC__ ;
