diff --git a/module/zfs/zfs_osx.cpp b/module/zfs/zfs_osx.cpp
index f05eb5f..e43503d 100644
--- a/module/zfs/zfs_osx.cpp
+++ b/module/zfs/zfs_osx.cpp
@@ -22,7 +22,7 @@ extern "C" {
   extern kern_return_t _stop(kmod_info_t *ki, void *data);
 
 };
-  __attribute__((visibility("default"))) KMOD_EXPLICIT_DECL(net.lundman.zfs, "1.0.0", _start, _stop)
+  __attribute__((visibility("default"))) KMOD_EXPLICIT_DECL(net.lundman.zfs, "5.0.8", _start, _stop)
   __private_extern__ kmod_start_func_t *_realmain = 0;
   __private_extern__ kmod_stop_func_t  *_antimain = 0;
   __private_extern__ int _kext_apple_cc = __APPLE_CC__ ;
diff --git a/META b/META
index 4d5437b..0003d29 100644
--- a/META
+++ b/META
@@ -1,7 +1,7 @@
 Meta:         1
 Name:         zfs
 Branch:       1.0
-Version:      1.3.0
+Version:      1.3.1
 Release:      1
 Release-Tags: relext
 License:      CDDL
