--- distribution.dist	2018-03-06 06:53:48.000000000 -0800
+++ distribution.dist	2018-03-06 06:55:07.000000000 -0800
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<installer-gui-script authoringTool="Packages" authoringToolVersion="1.2.2" authoringToolBuild="316" minSpecVersion="1.0">
+<installer-gui-script authoringTool="Packages" authoringToolVersion="1.2.2" authoringToolBuild="316" minSpecVersion="2">
     <options rootVolumeOnly="true" customize="never" allow-external-scripts="true"/>
-    <installation-check script="installation_check()"/>
+    <volume-check>
+        <allowed-os-versions>
+            <os-version min="10.12.0"/>
+        </allowed-os-versions>
+    </volume-check>
+    <installation-check script="installation_check()">
+        <ram min-gb="2.00"/>
+    </installation-check>
     <!--+==========================+
         |       Presentation       |
         +==========================+-->
@@ -233,4 +240,4 @@
 	}
 
     </script>
-</installer-gui-script>
\ No newline at end of file
+</installer-gui-script>
