--- distribution.dist	2014-07-22 00:26:31.000000000 -0700
+++ distribution.dist	2014-07-21 23:49:57.000000000 -0700
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <installer-script authoringTool="Packages" authoringToolVersion="1.1.1" authoringToolBuild="2B71" minSpecVersion="1.0">
     <options rootVolumeOnly="true" customize="never" allow-external-scripts="true"/>
+    <volume-check>
+        <allowed-os-versions>
+            <os-version min="10.8.0"/>
+        </allowed-os-versions>
+    </volume-check>
     <installation-check script="installation_check()"/>
     <!--+==========================+
         |       Presentation       |
