--- distribution.dist	2018-10-02 17:04:57.000000000 +0900
+++ distribution.dist	2018-10-02 17:06:12.000000000 +0900
@@ -1,7 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<installer-gui-script authoringTool="Packages" authoringToolVersion="1.2.3" authoringToolBuild="373" minSpecVersion="1.0">
+<installer-gui-script authoringTool="Packages" authoringToolVersion="1.2.3" authoringToolBuild="373" minSpecVersion="2">
     <options rootVolumeOnly="true" customize="never" allow-external-scripts="true"/>
-    <installation-check script="installation_check()"/>
+    <volume-check>
+        <allowed-os-versions>
+            <os-version min="10.14.0"/>
+        </allowed-os-versions>
+    </volume-check>
+    <installation-check script="installation_check()">
+        <ram min-gb="2.00"/>
+    </installation-check>
     <!--+==========================+
         |       Presentation       |
         +==========================+-->
