--- distribution.dist	2014-08-10 03:09:22.000000000 -0700
+++ distribution.dist	2014-08-10 03:09:30.000000000 -0700
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<installer-script authoringTool="Packages" authoringToolVersion="1.1.2" authoringToolBuild="2B104" minSpecVersion="1.0">
+<installer-script authoringTool="Packages" authoringToolVersion="1.1.2" authoringToolBuild="2B104" minSpecVersion="2">
     <options rootVolumeOnly="true" customize="never" allow-external-scripts="true"/>
-    <installation-check script="installation_check()"/>
+    <installation-check script="installation_check()">
+        <ram min-gb="2.00"/>
+    </installation-check>
     <!--+==========================+
         |       Presentation       |
         +==========================+-->
